src/GrassPokemon.java

```diff
@@ -0,0 +1,45 @@
+public class GrassPokemon extends Pokemon{
+    private String damageType;
+    private String effective;
+
```

Comment:
Mooi gebruik gemaakt van verschillende variabele in je subklassen. Hierdoor hebben je subklassen bestaansrecht en staan je variabele op het juiste niveau